modules/python/python3/extra_modules/misc/__init__.py

```diff
@@ -0,0 +1 @@
+from .version import get_ocv_version
```

Comment:
> modules/python/python3/extra_modules

It makes sense to move extra modules to the `package` location instead of `python3`

https://github.com/opencv/opencv/tree/master/modules/python/package